datanode/gateway/graphql/marshallers/ledgerentry_field.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022 Gobalsky Labs Limited
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE.DATANODE file and at https://www.mariadb.com/bsl11.
+//
+// Change Date: 18 months from the later of the date of the first publicly
+// available Distribution of this version of the repository, and 25 June 2022.
+//
+// On the date above, in accordance with the Business Source License, use
+// of this software will be governed by version 3 or later of the GNU General
+// Public License.
+
+package marshallers
+
+import (
+	"fmt"
+
+	v2 "code.vegaprotocol.io/vega/protos/data-node/api/v2"
+	"github.com/99designs/gqlgen/graphql"
+)
+
+func MarshalLedgerEntryField(*v2.LedgerEntryField) graphql.Marshaler {
+	panic("not implemented")
+}
+
+func UnmarshalLedgerEntryField(i interface{}) (*v2.LedgerEntryField, error) {
+	v, ok := i.(string)
+	if !ok {
+		return nil, fmt.Errorf("expected string in ledger entry field")
+	}
+
+	var lf v2.LedgerEntryField
+	switch {
+	case v == "TransferType":
+		lf = v2.LedgerEntryField_LEDGER_ENTRY_FIELD_TRANSFER_TYPE
+	default:
+		return nil, fmt.Errorf("unknown ledger entry field %v", v)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unknown ledger entry field %q", v)
```

Reply:
This, as well as 
https://github.com/vegaprotocol/vega/pull/6368#discussion_r992336390,
https://github.com/vegaprotocol/vega/pull/6368#discussion_r992336997,
https://github.com/vegaprotocol/vega/pull/6368#discussion_r992337806,
https://github.com/vegaprotocol/vega/pull/6368#discussion_r992338769,
https://github.com/vegaprotocol/vega/pull/6368#discussion_r992339322,
https://github.com/vegaprotocol/vega/pull/6368#discussion_r992341713
have been addressed